examples/lighting-app/lighting-common/lighting-app.matter

```diff
@@ -1144,53 +1144,6 @@ server cluster OccupancySensing = 1030 {
   readonly global attribute int16u clusterRevision = 65533;
 }
 
-client cluster OnOff = 6 {
```

Comment:
I think the idea was in fact to have an app that has both client and server lighting bits in the same app.  We've run into multiple SDK problems with that setup that had to be fixed, so this has been a useful smoketest.

Reply:
Interesting.. This looks pretty weird to a user though. Does all-clusters-app capture that case?

Reply:
FWIW I think for our purposes we would much rather have the lighting sample be only a light. This sample is supposedly an example of a smart light. I would expect it to just be a light bulb and light bulbs are not light switches.

Reply:
all-clusters-app doesn't really have client clusters.

I don't think it's necessarily a problem to remove the switch from lighting-app.  Just please check with whoever added it.

Reply:
I have reached out in the first comment on this PR.